catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/permissions/PermissionsResource.java

```diff
@@ -38,7 +37,7 @@
 public class PermissionsResource {
   private final Authorizer authorizer;
 
-  public PermissionsResource(CollectionDAO dao, @NonNull Authorizer authorizer) {
+  public PermissionsResource(@NonNull Authorizer authorizer) {
```

Comment:
This will result in runtime error  from CollectionRegistry to register this resource, since it looks for constructor with two args in the order Config and Authorizer

Reply:
https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/CollectionRegistry.java#L202

Reply:
Okay reverted that change with suppress unused warning annotation...